slrealizer/realize_om10.py

```diff
@@ -0,0 +1,298 @@
+#from __future__ import absolute_import
+#from __future__ import division
+#from __future__ import print_function
+
+from realize_sl import SLRealizer
+from utils.constants import *
+from utils.utils import *
+import numpy as np
+import galsim
+
+class OM10Realizer(SLRealizer):
+    
```

Comment:
This class needs a docstring. I forget whether it goes in the `__init__` method or here, you should check the numpydoc and /or sphinx docs. I guess we shoudl also read up on how to document subclasses.

Reply:
Added simple docstring!